docs/rfcs/RFC0009-OS-Packaging-Requirements.md

```diff
@@ -256,20 +256,177 @@ Prereleases:
+8.0.1        # Patch release
+8.0.2        # Patch release
+8.1.1        # Patch release
+
+Potential Extensions:
+8.1.0.1      # Point release (customer specific releases)
+```
+
+# Prereleases:
+```
+X.Y.Z-rcN
+
+N: release candidate no
+```
+
+# Nightly Releases:
+```
+X.Y.Z~YYYYMMDD
+```
+
+# Dev builds:
+```
+X.Y.Z~YYYYMMDDg<short-git-sha>   # git sha shortened to 8 bytes
+```
+# Package Version Ordering
+Package version in increasing order
+```
+8.1.0~20251201gf689a8e   # Development
+8.1.0~20251202ga123456   # Development
+8.1.0~20251203           # Nightly
+8.1.0~rc1                # Release Candidate 1
+8.1.0~rc2                # Release Candidate 2
+8.1.0                    # Final release
+8.1.1                    # Next patch release
+```
+
+# RPM package samples
+```
+rocm-8.1.0~20251201gf689a8e-1.x86_64.rpm                   # Development build using build no
+rocm-8.1.0~20251201gf689a8e-1234567.x86_64.rpm             # Development build using github runid
+rocm-8.1.0~20251203-1.x86_64.rpm                           # Nightly release
+rocm-8.1.0~rc1-1.x86_64.rpm                                # Release Candidate 1
+rocm-8.1.0~rc2-2345678.x86_64.rpm                          # Release Candidate 2 using github runid
+rocm-8.1.0-1.x86_64.rpm                                    # Final release
+rocm-8.2.0-3456789.x86_64.rpm                              # Next Minor release
+rocm8.1-8.1.0~20251201gf689a8e-1.x86_64.rpm                # Versioned Development build using build no
+rocm8.1-8.1.0~20251201gf689a8e-1234567.x86_64.rpm          # Versioned Development build using github runid
+rocm8.1-8.1.0~20251203-1.x86_64.rpm                        # Versioned Nightly release
+rocm8.1-8.1.0~rc1-1.x86_64.rpm                             # Versioned Release Candidate 1
+rocm8.1-8.1.0~rc2-2345678.x86_64.rpm                       # Versioned Release Candidate 2 using github runid
+rocm8.1-8.1.0-1.x86_64.rpm                                 # Versioned Final release
+rocm8.2-8.2.0-3456789.x86_64.rpm                           # Versioned Next Minor release
+```
+
+### Debian Package
+Debian packages should follow the file name format as follows
+
+```
+Non-Versioned package
+<package-name>_<package-version>-<release-version>.<arch>.deb
+
+Versioned package
+<package-name><package-version>_<package-version>-<release-version>_<arch>.deb
+
+  - <package-name>: The name of the package/module
+  - <package-version>: The version of the package. As defined in following session.
+  - <release-version>: Build no. Github run id should be used in case of TheRock build.
+  - <arch>: The target architecture (amd64).
+```
+
+## Debian Package Versioning Requirements
+
+# Stable Releases:
+```
+X.Y.Z
+```
+Stable Release Version Examples:
+```
+8.0.0        # Major release
+8.1.0        # Minor release
+8.0.1        # Patch release
+8.0.2        # Patch release
+8.1.1        # Patch release
+
+Potential Extensions:
+8.1.0-1      # Package rebuild (no code change)
```

Comment:
As for the RPM packages, the build number is not part of the package version  if I am not wrong and thus should not be mentioned here.

Reply:
Thanks for pointing out. Removed it